ui/src/utils/CommunityRuleMaps.tsx

```diff
@@ -0,0 +1,43 @@
+import { IRule } from '../components/Rules/Rule/RuleCreator'
+
+export enum Application {
+  PLEX,
+  RADARR,
+  SONARR,
+  OVERSEERR,
+  TAUTULLI,
+  JELLYSEERR,
+}
+
+export enum TVLevel {
+  SHOW = 'show',
+  SEASON = 'season',
+  EPISODE = 'episode',
+}
+
+export const applicationNames: Record<number, string> = {
```

Comment:
It's better to reference the enum as the key here. When Application is updated, this line will error until the new value is added. It's also just generally more readable.

```suggestion
export const applicationNames: Record<Application, string> = {
```